### PR TITLE
Add kuma 2.0 release with merbridge blog

### PR DESCRIPTION
**Pull request by kebe7jun**

### CODE REVIEW

File: content/zh/blog/2022/kuma-2.0-with-merbridge/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+![Kuma 2.0 eBPF 与 iptables 性能对比](./imgs/kuma-2.0-ebpf-vs-iptables.png)
+
+> We are utilizing the Merbridge OSS project within our eBPF capabilities and are very excited that we have been able to contribute back to that library and become co-maintainers. We look forward to working more with the Merbridge team as we continue to explore different areas to include eBPF functionality in Kuma.
+
+我们非常高兴，能够作为一个开源项目，为 Kuma 提供如此令人兴奋的能力，这意味着，基本不需要任何开销，即可降低网格应用通讯延迟！
+
+Kuma 从 6 月开始，就在着手于集成 Merbridge 项目，试图使用社区现有的能力来为 Kuma 提供加速能力。
+
+得益于 Merbridge 比较清晰的架构设计，Kuma 在很短的时间内就完成了对 Kuma 的兼容，非常感谢 Kuma 社区能够为 Merbridge 贡献如此重要的能力，帮助社区共同成长！
```
Comment (windsonsea) [inline content/zh/blog/2022/kuma-2.0-with-merbridge/index.md]:
```suggestion
得益于 Merbridge 比较清晰的架构设计，Kuma 在很短的时间内就完成了对 Merbridge 的兼容。非常感谢 Kuma 社区能够为 Merbridge 贡献如此重要的兼容能力，这有助于双方社区共同成长！
```

File: content/zh/blog/2022/kuma-2.0-with-merbridge/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+根据 Kuma 官方的描述，Kuma 正是使用了 Merbridge，来实现 eBPF 的能力。
+
+![Kuma 2.0 eBPF 与 iptables 性能对比](./imgs/kuma-2.0-ebpf-vs-iptables.png)
+
+> We are utilizing the Merbridge OSS project within our eBPF capabilities and are very excited that we have been able to contribute back to that library and become co-maintainers. We look forward to working more with the Merbridge team as we continue to explore different areas to include eBPF functionality in Kuma.
+
+我们非常高兴，能够作为一个开源项目，为 Kuma 提供如此令人兴奋的能力，这意味着，基本不需要任何开销，即可降低网格应用通讯延迟！
+
+Kuma 从 6 月开始，就在着手于集成 Merbridge 项目，试图使用社区现有的能力来为 Kuma 提供加速能力。
```
Comment (windsonsea) [inline content/zh/blog/2022/kuma-2.0-with-merbridge/index.md]:
```suggestion
Kuma 从 6 月开始，就在着手于集成 Merbridge 项目，尝试使用社区现有的能力来为 Kuma 提供加速能力。
```

File: content/zh/blog/2022/kuma-2.0-with-merbridge/index.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+![Kuma 2.0 发布预览](./imgs/kuma-2.0-release-preview.png)
+
+根据 Kuma 官方的描述，Kuma 正是使用了 Merbridge，来实现 eBPF 的能力。
+
+![Kuma 2.0 eBPF 与 iptables 性能对比](./imgs/kuma-2.0-ebpf-vs-iptables.png)
+
+> We are utilizing the Merbridge OSS project within our eBPF capabilities and are very excited that we have been able to contribute back to that library and become co-maintainers. We look forward to working more with the Merbridge team as we continue to explore different areas to include eBPF functionality in Kuma.
+
+我们非常高兴，能够作为一个开源项目，为 Kuma 提供如此令人兴奋的能力，这意味着，基本不需要任何开销，即可降低网格应用通讯延迟！
```
Comment (windsonsea) [inline content/zh/blog/2022/kuma-2.0-with-merbridge/index.md]:
```suggestion
我们非常高兴 Merbridge 作为一个开源项目，能够为 Kuma 提供如此令人兴奋的能力。这意味着，基本不需要任何开销，即可降低网格应用通讯延迟！
```

File: content/zh/blog/2022/kuma-2.0-with-merbridge/index.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+Kuma 从 6 月开始，就在着手于集成 Merbridge 项目，试图使用社区现有的能力来为 Kuma 提供加速能力。
+
+得益于 Merbridge 比较清晰的架构设计，Kuma 在很短的时间内就完成了对 Kuma 的兼容，非常感谢 Kuma 社区能够为 Merbridge 贡献如此重要的能力，帮助社区共同成长！
+
+截止目前，Merbridge 已经支持了 Istio、Linkerd2 和 Kuma 等主流的服务网格，也正在做很多事情，比如 v4/v6 双栈支持、Ambient Mesh 支持、更低的内核版本要求等。希望 Merbridge 能够被越来越广泛的应用，并且能够真实的帮助到大家。
```
Comment (windsonsea) [inline content/zh/blog/2022/kuma-2.0-with-merbridge/index.md]:
```suggestion
截止目前，Merbridge 已经支持了 Istio、Linkerd2 和 Kuma 等主流的服务网格，也计划了很多新的特性，比如 IPv4/IPv6 双栈支持、Ambient Mesh 支持、更低的内核版本要求等。希望 Merbridge 能够被越来越广泛的应用，并且能够真实地帮助到大家。
```

**Comment by kebe7jun** [comment]
> /lgtm

@windsonsea need using Github Approve.

**Review by windsonsea — APPROVED** [review]

**Pull request merged**